### PR TITLE
Calculator component

### PR DESCRIPTION
**Pull request by RafaelEchart**
In this milestone I did the following: 

- Deleted the default React logo and styles (or changed to my requirements)
- Created a directory called components
- Created a component called Calculator.js and import it into the main APP component

- One of the requirements was to use Class Components and leave the logic for the next project.
     - If you use **Class components** and do not add a **Constructor and a initial state** the linters will tell you to use Functional components
  
image

     - For using the **Class component** approach you need to initialize at least an empty state 
  
image

 ### Instructions to run the React app (the below codes should be enter on the terminal): 
    - npm install
    - npm run start
    - **Brower will open with app**

My design (It will improve over time): 

image

_Hope you like it!_

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
   return (
```
Comment (rukundoeric) [inline src/App.js]: - [x] <b>REQUIRED:</b> The project requires  `App` to be class-based component. Kindly do that to comply with project requirements.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @RafaelEchart, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] N/A

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/App.js (hunk 1 above)
Comment (RafaelEchart) [reply src/App.js]: I didn't even notice that by default React.js, makes the App component Functional!

Thanks for the tip! It's solved!

**Review by rukundoeric — APPROVED** [review]
Hi @RafaelEchart,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**